suites/pre-release/dcf/4.preSignedURLTest.js

```diff
@@ -37,25 +49,35 @@ async function fetchDIDLists(I) {
     ).then((res) => new Gen3Response(res));
 
     const projectAccessList = httpResp.body.project_access;
```

Comment:
What does this look like?
I think it would be better to just create a record with one of the acl/authz that you know you have access to, in a similar way to creating the fail record

Reply:
https://github.com/uc-cdis/gen3-qa/pull/770/files#r813063007